### PR TITLE
[bn] Localize content/bn/docs/tasks/tools/included/_index.md

### PR DESCRIPTION
**Pull request by Arindam200**
Signed-off-by: Arindam Majumder [[arindammajumder2020@gmail.com](mailto:arindammajumder2020@gmail.com)]
Localizes `content/bn/docs/tasks/tools/included/_index.md` to Bengali
This is an initial PR as may need some improvements!
Tracking issue: https://github.com/kubernetes/website/issues/31677

### CODE REVIEW

File: content/bn/docs/tasks/tools/included/_index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+শিরোনাম: "অন্তর্ভুক্ত টুলস"
+বিবরণ: "প্রধান kubectl-installs-*.md পেজগুলিতে স্নিপেট অন্তর্ভুক্ত করা হবে স্নিপেট।"
```
Comment (bishal7679) [inline content/bn/docs/tasks/tools/included/_index.md:3]:
```suggestion
বিবরণ: "প্রধান kubectl-installs-*.md পেজগুলিতে স্নিপেট গুলি অন্তর্ভুক্ত করতে হবে।"
```

**Review by bishal7679 — APPROVED** [review]
/lgtm
/cc @rajibmitra @Imtiaz1234

**Comment by mitul3737** [comment]
Hey @Arindam200  , Our approvers for Bengali localization are @rajibmitra  and @mitul3737  & @Imtiaz1234  . After creating a PR, feel free to knock us on Slack or mention us.

**Review by mitul3737 — APPROVED** [review]

**Comment by mitul3737** [comment]
tide/merge-method-squash

**Comment by Arindam200** [comment]
Ok sir I will do that!

**Pull request merged**